packages/uploadthing/test/__test-helpers.ts

```diff
@@ -36,15 +38,17 @@ export const it = itBase.extend<{ db: MockDbInterface }>({
 });
 
 export const s3Mock = vi.fn();
-export const fetchMock = vi.fn();
+export const staticAssetMock = vi.fn();
+export const utApiMock = vi.fn();
```

Comment:
i also tried making these `vi.spyOn(globalThis, "fetch")` but that didn't pick up half the calls...